yocto/export-oe-tests.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# vagrant-cookbook
+# Copyright (C) 2017 Pelagicore AB
+#
+# Permission to use, copy, modify, and/or distribute this software for
+# any purpose with or without fee is hereby granted, provided that the
+# above copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+# WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR
+# BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES
+# OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+# WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+# ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+# SOFTWARE.
+#
+# For further information see LICENSE
+#
+#
+# Usage: export-oe-tests.sh <yoctodir> <images>
+#
+# Source the environment and export existing oe-tests
+#
+
+YOCTO_DIR="$1"
+IMAGES="$2"
+
+echo "BitBake'ing $IMAGES in $YOCTO_DIR"
+
+# Set up bitbake environment
+cd "$YOCTO_DIR/sources/poky/"
+source oe-init-build-env ../../build
+
+# A positive exit code from now on is fatal
+set -e
+
+# Export oe-tests
+time bitbake $IMAGES -c testexport
```

Comment:
We usually write bitbake -c <command> <image> rather than the other way around, but that's just cosmetics.